config/playground/jobs/sample_data_tooling/refresh_deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: refresh-data-cronjob
+spec:
+  # Runs everyday at 1 am UTC
+  schedule: "0 1 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: refresh-data-job
+              image: huyaboo/demo
```

Comment:
please move this image to our ECR, and use the good name for your image. Thanks

Reply:
Done, the image name is `sample-data-tooling` and is listed as a public ECR repo.